emails/migrations/0052_20220807_add_phone_sub_fields_to_profile.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 3.2.15 on 2022-09-07 19:48
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        (
+            "emails",
+            "0051_add_email_size_per_day_and_num_email_forwarded_per_day_fields",
+        ),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="profile",
+            name="date_phone_subscription_checked",
+            field=models.DateTimeField(blank=True, null=True),
```

Comment:
Did you want to convert this one to column-level defaults as well?

Reply:
I thougth about it briefly ... I *think* this should work, right?

1. The `RelayNumber` stats fields will get their default values at the time the migration runs
2. This `date_phone_subscription_checked` gets a value set to the day that the migration runs

1 and 2 work out well together?

Reply:
makes sense 👍